site-specific/farrer-park/mm.py

```diff
@@ -1,3 +1,5 @@
+# ruff: noqa
```

Comment:
Is the idea to `noqa` all site specific content?

Reply:
No, just the ones that were failing that I don't want to have to deal with :)